mypy/typeanal.py

```diff
@@ -272,7 +275,7 @@ def try_analyze_special_unbound_type(self, t: UnboundType, fullname: str) -> Opt
             self.fail("Final can be only used as an outermost qualifier"
                       " in a variable annotation", t)
             return AnyType(TypeOfAny.from_error)
-        elif fullname == 'typing.Tuple':
+        elif fullname in ('typing.Tuple', 'builtins.tuple'):
```

Comment:
As with `builtins.type` `builtins.tuple` is only allowed if you're using py39+ or `from __future__ import annotations` is used.
```suggestion
        elif (fullname == 'typing.Tuple' or
             (fullname == 'builtins.tuple' and (
                 self.options.python_version >= (3, 9) or self.api.is_future_flag_set('annotations')))):
```

Reply:
Of course. Thanks.